### PR TITLE
timew: add page

### PR DESCRIPTION
**Pull request by abactel**
This will add the `timew` command. Timewarrior is advertised as:

> ... a command line time tracking application, which allows you to record time spent on activities.

[For more information see their website](https://taskwarrior.org/docs/timewarrior/)

### CODE REVIEW

File: pages/common/timew.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# timew
+
+> Time tracker.
+
+- Start a new stopwatch:
+
+`timew {{item_to_time}} start`
```
Comment (agnivade) [inline pages/common/timew.md]: What is "item_to_time" here ? The man [page](https://taskwarrior.org/docs/timewarrior/syntax.html) doesn't mention anything before the 'start' option.

Comment (abactel) [inline pages/common/timew.md]: The `{{item_to_time}}` is supposed to be the tag, an identifier for the thing the user is tracking. I didn't realize the man page showed the tag being placed after an initial `timew start`, as both work methods work equally. I've changed it in commit 2b48f56 to match the man page anyway.

**Comment by agnivade** [comment]
@abactel - I included the word 'tag' itself as I think it fits better with the description. Let me know what you think of the changes.

**Comment by abactel** [comment]
I agree with this, these changes make a lot of sense

**Comment by agnivade** [comment]
@jeeftor - please check

**Comment by sbrl** [comment]
What about stopping a single timer?

**Comment by abactel** [comment]
I forgot about that. The command to stop a single stopwatch of a given tag is `timew stop activity_tag`. I'll create a commit to add this now.

**Pull request closed without merging**